manuscript.md

```diff
@@ -1,218 +1,236 @@
-sets. Applying this model to the entire dataset (including species pairs never
-observed co-occuring in the dataset) identified 1831 new possible interactions
--- 382 of which were in pairs of species never considered prior. This suggests
-that meaningful information about ecological interactions is structured within
-network data. Now, the question becomes: home do we make our prediction of
-interaction networks _better_?
+learning framework [@Innes2018FluEle].
+
+We first aggregate all interactions into a cooccurence matrix $C$ which
+represents whether a given pair of species $(i,j)$ was observed coexisting
+across any location [@Dunne2006NetStr]. We then transform this cooccurence
+matrix $C$ via probabilistic PCA [@Tipping1999ProPri] and use the first 15
+values from this PCA as the features vector for each species $i$. For each pair
+of (host, parasite) species $(i,j)$, we then feed the features vectors $[v_i,
+v_j]$ into a neural network. The neural-network uses four feed-forward layers
+(the first $\text{RELU}$, the rest $\sigma$) with appropriate dropout rates
+($0.5$). This produces an output layer which is the probability-score for
+interaction between species $i$ and $j$.
+
+We then train this neural network by dividing the original dataset into
+test and training sets.
+During the training of this neural network, the batches of 16 items used for
+training were constrained to have at least 10 positive interactions, as
+@timpreprintcite show slightly inflating the dataset with
+positive interactions enables us to counterbalance sampling biases.
+Validating this model on the test data shows our model provides highly effective
+prediction of interactions between pairs of species not present in the original data (@fig:example).
+
+![Proof-of-Concept: An empirical network
+[from @Hadfield2014TalTwo] is converted intro latent features using
+probabilistic PCA, then used to train a deep neural network to predict
+species interactions. The initial and imputed networks are represented
+as their tSNE embedding, and the colors of nodes are the cluster to
+which they are assigned based on a $k$-means clustering of the tSNE
+output.](figures/figure1.png){#fig:example}
+
+
+This case study shows that a simple neural network can be very effective in
+predicting species interactions even without additional data about each species.
+Applying this model to the entire dataset (including species pairs never
+observed cooccuring) identified N new possible interactions -- M of which were
+in pairs of species never considered prior. This model reaches similar levels of
+predictive efficacy as previous studies that use far more species-level data and
+mechanistic assumptions [@Gravel2013InfFoo], which serves to highlight the
+potential for including external sources of data  for improving our prediction
+of interaction networks.
+
+However, we later found trait data. This emphasizes the need for open data
```

Comment:
On the fence about how I feel about this statement...

Reply:
that a placeholder for a more clear discussion re #131